tests/test_spinnaker.py

```diff
@@ -0,0 +1,25 @@
+import acquire
+import pytest
+from acquire import DeviceKind
+
+
+@pytest.fixture(scope="module")
+def _runtime():
+    runtime = acquire.Runtime()
+    yield runtime
+
+
+@pytest.fixture(scope="function")
+def runtime(_runtime: acquire.Runtime):
+    yield _runtime
+    _runtime.set_configuration(acquire.Properties())
+
+
+def test_blackfly_camera_is_preset(runtime: acquire.Runtime):
```

Comment:
Present or preset?

Reply:
This is copy-pasted from `test_egrabber.py`, but I think this should be present instead of preset, so I changed it in both places.